### ISSUE TITLE
Idle timeout message happened twice

### ISSUE THREAD

**Issue by fndax** [lykoss/lykos#506]
miney was unactivated vg. They didn't use !swap at all that game I think.

```
20:07:44 *** minetester (~mineteste@user/minetester) [Minetester] NoName has joined the channel 
[...]
20:10:05 *** +miney (~mineteste@user/minetester) has quit (Ping timeout: 268 seconds)
20:10:05 <@lykos> miney has gone missing.
[...]
20:11:10 <@lykos> miney didn't get out of bed for a very long time and has been found dead. The survivors bury the vengeful ghost's body.
20:11:10 <@lykos> miney was mauled by wild animals and has died. It seems that vengeful ghost meat is tasty.
```

**Comment by MishaCatskill** [comment]
this also applied two warnings, one for idling out and one for quitting IRC

**Comment by fndax** [comment]
lykos apparently applied one warning for idle out and one for quitting. miney was also gunner.

**Comment by fndax** [comment]
20:18:52 <+moonmoon> the timer for the /quit grace period and the timer for regular idle out (not talking for 5 mins) both fired at the same time
20:19:28 <+moonmoon> we should check if they're already dead once we acquire the rlock
20:19:30 <+moonmoon> and abandon if they are

**Comment by skizzerz** [comment]
Fixed in 73adae88ec1bb6fd43463b4e13ab5c3541297f1d
